dashboards/google-dns-query/metadata.yaml

```
@@ -0,0 +1,10 @@
+sample_dashboards: 
+  - 
+    category: DNS Query
+    id: dns-query
+    display_name: Cloud DNS
+    description: |-
+      This dashboard has charts displaying DNS query metrics.
```

```suggestion
      This dashboard has charts displaying DNS query metrics, including Error Rate, QPS, and 99th percentile latency.
```

Let's keep this consistent with other dashboard descriptions by briefly describing the metrics as well.